docs/pages/concepts/architecture/meshsync.md

```diff
@@ -28,7 +28,15 @@ The resources that are present inside the cluster are discovered efficiently wit
 
 ### Listening to events/changes on every component
 
-MeshSync implements several informers/listeners across each resource to listen to changes that occur to them. These are reflected at real time and are updated in their respective areas.
+The informer in MeshSync actively listens to changes in resources and updates them in real time based on the provided informer 
+configuration in the CRD.
 
 ## MeshSync FAQs
 
+### How to reconfigure Meshsync informer?  
+MeshSync is managed by [Meshery Operator]({{site.baseurl}}/concepts/architecture/operator), which watches for changes on MeshSync CRD for 
+changes and updates the deployed resource accordingly. You can update the CRD using kubectl through the following steps
+- Download the CRD with `kubectl get crd meshsyncs.meshery.layer5.io -o yaml > meshsync.yaml`
+- Open the downloaded file and edit the field `informer_config` to blacklist all the types of resources that you don't want updates from.
+- Apply the new definition with `kubectl apply -f meshsync.yaml` 
+
```

Comment:
```suggestion
### How to configure MeshSync's resource discovery behavior: Can specific, "uninteresting" resources be blacklisted?  

MeshSync is managed by [Meshery Operator]({{site.baseurl}}/concepts/architecture/operator), which watches for changes on the `meshsync` CRD for changes and updates the deployed MeshSync instance accordingly. You can blacklist specifi Kubernetes resources from being discovered and watched by MeshSync. In order to identify the list of one or more resources for MeshSync to ignore, update the `meshsync` CRD using kubectl:

- Download the CRD with `kubectl get crd meshsyncs.meshery.layer5.io -o yaml > meshsync.yaml`
- Open the downloaded file and edit the field `informer_config` to blacklist all the types of resources that you don't want updates from.
- Apply the new definition with `kubectl apply -f meshsync.yaml` 

```

This FAQ needs a specific example that includes multiple resource names in the blacklist.

Reply:
suggestion has a "specifi" instead of "specific"

Reply:
Good eye. I've corrected the misspelling. 👍